src/desktop/apps/artist2/client.js

```diff
@@ -4,12 +4,13 @@ import styled from 'styled-components'
 import { buildClientApp } from 'reaction/Router'
 import { routes } from 'reaction/Styleguide/Pages/Artist/routes'
 import { data as sd } from 'sharify'
+import mediator from 'desktop/lib/mediator.coffee'
 
 buildClientApp({ routes, user: sd.CURRENT_USER })
   .then(({ ClientApp }) => {
     ReactDOM.hydrate(
       <Container>
-        <ClientApp />
+        <ClientApp force={{ mediator }} />
```

Comment:
`force` seems like it could be an overloaded term, maybe `webClient` or something?

Reply:
I was thinking the same, should avoid tying things to system names. Was gonna think about this today.